docs/agents/sematext-agent/packages/configuration.md

```diff
@@ -0,0 +1,75 @@
+title: Configuring Package Collection
+description: Package Collection is enabled by default in the Sematext Agent configuration.
+
+Package Collection is enabled by default in the [Sematext Agent configuration](../agents/sematext-agent/containers/configuration/). It collects packages from two different sources:
+
+- packages installed on virtual/bare-metal hosts
+- packages residing inside containers
+
+You can *disable* package collection by setting the following properties:
+
+<div class="mdl-tabs mdl-js-tabs mdl-js-ripple-effect">
+ <div class="mdl-tabs__tab-bar">
+     <a href="#file-pkg-enabled" class="mdl-tabs__tab is-active">Config file</a>
+     <a href="#env-pkg-enabled" class="mdl-tabs__tab">Env variable</a>
+ </div>
+
+ <div class="mdl-tabs__panel is-active" id="file-pkg-enabled">
+   <pre>
+   # st-agent.yml
+   pkg:
+     enabled: false
+     container-enabled: false
+   </pre>
+ </div>
+ <div class="mdl-tabs__panel" id="env-pkg-enabled">
+   <pre>
+   PKG_ENABLED=false
+   PKG_CONTAINER_ENABLED=false
+   </pre>
+ </div>
+</div>
+
+Additionally, you can tweak the interval that determines how often packages are scanned. By default, the interval is every 10 minutes. The agent retrieves packages from the file system and any running containers, then compares the current state to decide which packages should be reported as installed or removed, and finally ships the events to Sematext Cloud. Asides from that, the agent proactively reacts to file system changes and bulks the packages in real time.
```

Comment:
Asides => Aside (probably a mixup with "Besides that")

> bulks the packages

Hm, what does this mean?  It's probably referring to sending data in bulk, but what would it mean to bulk packages?

Reply:
If user installs 10 packages, the agent will capture them as soon as there are written to the file system, store to intermediate buffer and flush immediately instead of waiting for the next iteration that happens each 10 minutes.

Reply:
Aha.
Asides from that, the agent proactively reacts to file system changes and bulks the packages in real time.
=>
In addition to reading package info periodically, the agent also captures package install, update, and uninstall events and ships them in real time.